__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const request = require('supertest');
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+
+afterAll(() => db.end());
+/* beforeEach(() => seed(data)); */
```

Comment:
is there a specific reason why this is commented out? If not can you put it back in please